Example/Example Swift/MultipageDocumentService.swift

```diff
@@ -27,6 +27,7 @@ final class MultipageDocumentsService: DocumentServiceProtocol {
             .map { $0.value }
             .sorted()
             .map { $0.info }
+            .filter { $0.documentUrl != nil}
```

Comment:
If they don't have a `documentUrl`, they are not uploaded yet?

Reply:
Exactly, that it might change if I find a solution or the pending handler.